### DISCUSSION TITLE
Cap mob catch up speed from TPS catchup

### DISCUSSION THREAD

**Discussion by aikar** [PaperMC/Paper#10488] [Ideas]
When TPS catchup kicks in from a large spikes, mobs run at super speed. see if we can mark the tick as hasnt had a delay, and run mobs as inactive ticks to improve experience on them not rushing and killing players.

**Comment by Drc-DEV** [comment]
Please I really need this fix

**Comment by lynxplay** [comment]
For now I re-labeled this as a feature issue instead of a bug. This is technically vanilla behavior, changing this for user experience is a plausible goal, tho not particularly something considerable as a bug.

Concerning it has not been touched/worked on since 2018, I marked this as a general low priority too.
Will be revisited in regards to feasibility at a later date :+1:
